Content.Client/Hands/Systems/HandsSystem.cs

```diff
@@ -240,9 +243,9 @@ public void UIHandOpenContextMenu(string handName)
                 return;
             }
 
-            _verbs.VerbMenu.OpenVerbMenu(entity);
+            _ui.GetUIController<VerbMenuUIController>().OpenVerbMenu(entity);
```

Comment:
isnt the _verbs dependency here now useless?

Reply:
that's for verb system not the UI controller

Reply:
o wait i see ur point, yes it is